util/log.c

```diff
@@ -73,8 +73,8 @@ static void qemu_logfile_free(QemuLogFile *logfile)
 
 static bool log_uses_own_buffers;
 
-__attribute__((weak)) void flush_tcg_on_log_instr_chage(void);
-__attribute__((weak)) void flush_tcg_on_log_instr_chage(void) {
+__attribute__((weak)) void qemu_log_instr_flush_tcg(void);
+__attribute__((weak)) void qemu_log_instr_flush_tcg(void) {
     // Real implementation in translate-all.c
```

Comment:
`// Real implementation in accel/tcg/log_instr.c since we can't use invoke tcg functions here`